master/internal/workspace/postgres_workspace.go

```diff
@@ -132,3 +133,41 @@ func AllWorkspaces(ctx context.Context) ([]*model.Workspace, error) {
 	}
 	return w, nil
 }
+
+// GetNamespaceFromWorkspace returns the namespace for the given workspace and cluster.
+func GetNamespaceFromWorkspace(ctx context.Context, workspaceName string, clusterName string) (string, error) {
+	var ns string
+	err := db.Bun().
+		NewSelect().
+		TableExpr("workspaces as w").
+		ColumnExpr("n.namespace_name").
+		Join("JOIN workspace_namespace_bindings AS n ON  n.workspace_id = w.id").
+		Where("w.name = ? and n.cluster_name = ?", workspaceName, clusterName).
+		Scan(ctx, &ns)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to get namespace")
+	}
+	return ns, nil
+}
+
+// GetAllNamespacesForRM gets all namespaces associated with a particular cluster. defaultNs is an optional
+// parameter, if there is no defaultNs provided, the "default" namespace will be added to the list instead.
+func GetAllNamespacesForRM(ctx context.Context, rmName string, defaultNs string) ([]string, error) {
+	var ns []string
+	err := db.Bun().
+		NewSelect().
+		Table("workspace_namespace_bindings").
+		ColumnExpr("DISTINCT namespace_name").
+		Where("cluster_name = ?", rmName).
+		Scan(ctx, &ns)
+	if err != nil {
+		return ns, errors.Wrapf(err, "failed to get namespace")
```

Comment:
use fmt.Errorf instead of errors.Wrapf: https://hpe-aiatscale.atlassian.net/wiki/spaces/ENGINEERIN/pages/1389035546/Go+Style+at+Determined#Error-Wrapping

Reply:
fixed!